.rubocop_todo.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Is very common in this module
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
```

Comment:
I had to disable this from time to time, it does not work well with large code base. Maybe we can opt-out by default?